src/extrovermectin.ts

```diff
@@ -219,9 +229,22 @@ function initializeDireWarren(): void {
   } while ("fluffy bunny" !== get("lastEncounter"));
 }
 
-export function intializeExtrovermectinZones(): void {
+export function initializeExtrovermectinZones(): void {
   if (get("beGregariousFightsLeft") === 0) {
     if (hasMonsterReplacers()) initializeCrates();
     initializeDireWarren();
   }
 }
+
+export function ponderPrediction(location: Location): Monster | null {
+  visitUrl("inventory.php?ponder=1", false);
```

Comment:
Perhaps cache this if no turns has passed?

Reply:
I plan on moving this to libram's upcoming CrystalBall.ponder function soon, which deliberately doesnt' cache it because you could have a free encounter that doesn't take a turn. So that micro-optimization would last at most a few days before it gets rolled back, unfortunately.